<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisorAttackClass</class>
 <widget class="QMainWindow" name="VisorAttackClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FRAV view</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pBShowImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QLabel" name="lImg_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>1001</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 213, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBShowImg_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBShowImg_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>is attack</string>
    </property>
   </widget>
   <widget class="QLabel" name="lAttack">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>ATTACK</string>
    </property>
   </widget>
   <widget class="QLabel" name="lBonaFide">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BONA FIDE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="visorAttack.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pBShowImg</sender>
   <signal>clicked()</signal>
   <receiver>VisorAttackClass</receiver>
   <slot>view()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBShowImg_4</sender>
   <signal>clicked()</signal>
   <receiver>VisorAttackClass</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBShowImg_5</sender>
   <signal>clicked()</signal>
   <receiver>VisorAttackClass</receiver>
   <slot>isAttack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>view()</slot>
  <slot>play()</slot>
  <slot>record()</slot>
  <slot>stop()</slot>
  <slot>selectFile()</slot>
  <slot>isAttack()</slot>
 </slots>
</ui>
